### DISCUSSION TITLE
Moonlight and LXQT

### DISCUSSION THREAD

**Discussion by billdotmu** [forkymcforkface/RGBPi-Extra#20] [General]
Hi. The now-deleted RGBPi-Extra repository had Moonlight and LXQT in separate folders. Will you be including those in this fork as well? Thanks.

**Comment by forkymcforkface** [comment]
those are in their own repo now https://github.com/forkymcforkface/RGBPi-Ports

**Reply by billdotmu** [reply]
Great, thank you!
